airstack-modules/modules/airstack/social/Readme.md

```diff
@@ -100,7 +100,18 @@
     renewalCost: BigInt,                       #renewal cost of the profile
   )
 ```
-
+```
+8. Track default profile change transaction
+  trackSocialUserDefaultProfileChange(
+    block: ethereum.Block,                     #ethereum block of the profile ownership change transaction
+    transactionHash: string,                   #transaction hash of the profile ownership change transaction
+    logOrCallIndex: BigInt,                    #log or call index - used to differentiate between multiple logs or calls in a single transaction
+    from: string,                              #intiator of the transaction
+    to: string,                                #contract address
+    tokenId: string,                           #token id of the profile token
+    tokenAddress: string,                      #token address of profile token
+    socialUserId: string                       #address of user who's default profile is set
```

Comment:
social user id is not necessarily an address, please update with ` #user id from respective dapp`